docs/mddocs/Quickstart/npu_quickstart.md

```diff
@@ -82,19 +82,42 @@ With the `llm-npu` environment active, use `pip` to install `ipex-llm` for NPU:
 conda activate llm-npu
 
 pip install --pre --upgrade ipex-llm[npu]
+
+:: [optional] for Llama-3.2-1B-Instruct & Llama-3.2-3B-Instruct
+pip install transformers==4.45.0 accelerate==0.33.0
+
+:: [optional] for glm-edge-1.5b-chat & glm-edge-4b-chat
+pip install transformers==4.47.0 accelerate==0.26.0
 ```
 
 ## Runtime Configurations
 
-For `ipex-llm` NPU support, set the following environment variable with active `llm-npu` environment:
+For `ipex-llm` NPU support, please set the following environment variable with active `llm-npu` environment based on your device:
 
-```cmd
+<details>
+
+<summary>For Intel Core™ Ultra Processers (Series 2) with processor number 2xxV</summary>
+
+```bash
 set BIGDL_USE_NPU=1
 
-:: [optional] for MTL support
+# [optional] for Intel Core™ Ultra 5 Processor 228V & 226V
+set IPEX_LLM_NPU_DISABLE_COMPILE_OPT=1
+```
+
+</details>
+
+<details>
+
+<summary>For Intel Core™ Ultra Processers (Series 1) with processor number 1xxH</summary>
+
+```bash
+set BIGDL_USE_NPU=1
 set IPEX_LLM_NPU_MTL=1
 ```
 
+</details>
```

Comment:
I think there is no need to hide the configurations. We could use the format as we did in GPU qucikstart: https://github.com/intel-analytics/ipex-llm/blob/main/docs/mddocs/Overview/install_gpu.md#runtime-configuration-1

- For **xxxx**
   ```cmd
   ```
- For **xxxx**
   ```cmd
   ```

And we should use `bash` -> `cmd`?